lapce-data/src/data.rs

```diff
@@ -228,12 +245,40 @@ impl LapceData {
         }
 
         if let Some((window_id, _)) = windows.iter().next() {
-            for file in files {
+            for (i, file) in files.iter().enumerate() {
+                let line_column = line_columns[i];
+                let widget_id = Arc::try_unwrap(
+                    windows.get(window_id).unwrap().active_id.clone(),
+                )
+                .unwrap();
+
+                // open the file
                 let _ = event_sink.submit_command(
                     LAPCE_UI_COMMAND,
                     LapceUICommand::OpenFile(file.to_path_buf(), false),
                     Target::Window(*window_id),
                 );
+
+                // jump to line and column
+                let _ = event_sink.submit_command(
+                    LAPCE_UI_COMMAND,
+                    LapceUICommand::JumpToLineColLocation(
+                        Some(widget_id),
+                        EditorLocation {
+                            path: file.to_path_buf(),
+                            position: Some(line_column), // line info is included in column variable
+                            scroll_offset: Some( 
+                                Vec2 {
+                                    x: line_column.column as f64, 
+                                    y: line_column.line as f64,
+                                } 
+                            ),
```

Comment:
Doesn't this mean that if `position = None` then it is automatically assumed that `position = 1`? And similarly with `scroll_offset`?